.travis.yml

```diff
@@ -96,32 +102,44 @@ matrix:
       env:
         - MB_PYTHON_VERSION=3.6
         - NP_BUILD_DEP=numpy==1.11.3
+        - DAILY_BUILD=true
     - os: osx
       language: generic
       env:
         - MB_PYTHON_VERSION=3.7
         - NP_BUILD_DEP=numpy==1.14.5
         - NP_TEST_DEP=numpy==1.14.5
+        - DAILY_BUILD=true
 
 before_install:
-    - BUILD_DEPENDS="$NP_BUILD_DEP cython==0.28.4 $SCIPY_BUILD_DEP"
-    - TEST_DEPENDS="$NP_TEST_DEP pytest $SCIPY_TEST_DEP"
-    - source multibuild/common_utils.sh
-    - source multibuild/travis_steps.sh
-    - before_install
+  - if [ "$TRAVIS_BRANCH" == "master" ]; then
+    CONTAINER="pre-release";
+    BUILD_COMMIT=${DAILY_COMMIT:-$BUILD_COMMIT};
+    if [ "$DAILY_BUILD" == "false" ]; then
+    exit 0;
```

Comment:
maybe use indentation to make this multiline `before_install` bash thingy more readable. I think indentation does not break yaml: https://github.com/cloudpipe/cloudpickle/blob/master/.travis.yml#L52-L56